spec-dtslint/operators/window-spec.ts

```diff
@@ -6,5 +6,10 @@ it('should infer correctly', () => {
 });
 
 it('should enforce types', () => {
-  of(1).pipe(window('')); // $ExpectError
```

Comment:
Since `String` [implements](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/@@iterator) `Iterable` interface and since `Iterable` is declared in `ObservableInput`, `window` now accepts string as a valid input type. I didn't want to move this test to `'should infer correctly'` section so I don't confuse anyone reading these tests.